src/App/Part.cpp

```diff
@@ -109,6 +109,23 @@ PyObject *Part::getPyObject()
     return Py::new_reference_to(PythonObject);
 }
 
+void Part::handleChangedPropertyType(Base::XMLReader &reader, const char *TypeName, App::Property *prop)
+{
+    // Migrate Material from App::PropertyMap to App::PropertyLink
+    if (!strcmp(TypeName, "App::PropertyMap")) {
+        App::PropertyMap v;
```

Comment:
Please no single-character variables. This is hard to read for others. Please use a self-describing name.

Reply:
> In general I would prefer when you keep the handleChangedPropertyType routine like it is done in many other places in part and PartDesign.

Next time, would you be so kind to inform me beforehand? I did not know your expectations, hence I stole my code here: https://github.com/FreeCAD/FreeCAD/blob/2c072c70929bb359853bfa09a5b34977c8a0670b/src/Mod/Mesh/App/FeatureMeshSolid.cpp#L75

> Please no single-character variables. This is hard to read for others. Please use a self-describing name.

No problem, I fully agree. I'm gonna correct it.

Reply:
> Next time, would you be so kind to inform me beforehand? I did not know your expectations, hence I stole my code here:
> 
> https://github.com/FreeCAD/FreeCAD/blob/2c072c70929bb359853bfa09a5b34977c8a0670b/src/Mod/Mesh/App/FeatureMeshSolid.cpp#L75

that would also be fine. I objected the warning and the code you took also has no warning. 

In general, I am a new merger. Of course everybody has his preferences. Since I am new I try to keep the code as we have it in other places when possible, thus my objection.

I could also not inform you earlier since I took over the PR to review - 11 months after you made it ;-)